### ISSUE TITLE
Yellow warnings showing up in engine breaking (red) errors

### ISSUE THREAD

**Issue by GodotIsAMess** [godotengine/godot#9113]
This is under 2.1.3 stable:

![image](https://user-images.githubusercontent.com/23731618/26999445-949fe390-4d4e-11e7-8602-c78a1a79e5bd.png)

This is a warning, not an error. The engine already differentiates between the two because of the YELLOW and RED color squares. Yellow warnings shouldn't be shown as errors here:

![image](https://user-images.githubusercontent.com/23731618/26999462-032b372e-4d4f-11e7-995b-3529a88f514c.png)

This is just silly and scares the developer of an actual issue when that warning from the libpng library is already useless anyway.

Errors that have the red square square should only show up in the parenthesis ()

I have hundreds of png files. You want me to run a program to clean up the sRGB profile on them? After every time I edit a spritesheet as well? And you want the developer to see this as a game engine "ERROR" when previewing the scene? No. These sRGB incorrect profile errors should not show up in parenthesis () and should only show up, under the errors tab.

**Comment by kubecz3k** [comment]
I don't think amount of warning should not be there at all, usually they are referring to real programmer mistakes that can still run good only because godot is doing some checks. Sometimes it's not working this way, but that usually means we have some bug in Godot. 
Not sure what we should do with that incorrect sRGB warning. However what I was usually doing as a programmer if I had some assets with this concrete warnings was run some shell commands to fix them all (https://stackoverflow.com/questions/22745076/libpng-warning-iccp-known-incorrect-srgb-profile)

**Comment by GodotIsAMess** [comment]
Yeah, I think the error in yellow should not show in parenthesis under "Errors (xx)", as that button references **red squares** errors that are most likely _fatal_. 

Not to be argumentative and I appreciate the input, but the fix for the sRGB errors with a 3rd party tool is extremely silly. This means, every time a developer edits a spritesheet (.psd) to add a new image in Photoshop, the developer now has to run the image through a 3rd party tool.

And if the developer has hundreds of spritesheets or images to use within the game, they will continue to see those sRGB errors as a fatal one. Then when they click on the "Errors (xx)", it's just a warning.  Then the cycle continues, another dev will create this type of github request.

**Comment by vnen** [comment]
I think it should show the amount of warnings too. One option is to add a filter on the Errors tab to show only the errors or warnings or both.

But also, Photoshop (and other image editing tools) should be able to export PNG with a correct sRGB profile, so you need to file a bug report for them too.
